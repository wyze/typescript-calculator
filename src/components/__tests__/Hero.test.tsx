import { h } from 'preact'
import Hero from '../Hero'
import snapshot from './utils/snapshot'

describe('<Hero />', () => {
  it('renders', () => snapshot(<Hero />))
})
