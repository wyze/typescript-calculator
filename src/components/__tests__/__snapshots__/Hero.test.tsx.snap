// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> renders 1`] = `
.glamor-0,
[data-glamor-0] {
  background: #dedede;
  border-radius: 5px;
  display: block;
  margin-bottom: 4em;
  padding: 1em;
  text-align: center;
  width: 30em;
}

@media only screen and (min-width: 53.125em) {
  .glamor-0,
  [data-glamor-0] {
    width: 35em;
  }
}

<div
  className="glamor-0"
>
  <Title />
  <Features />
</div>
`;
